addon/plugins/links/index.ts

```diff
@@ -0,0 +1,25 @@
+import { PNode, ProsePlugin, RdfaEditorView } from '@lblod/ember-rdfa-editor';
+export const linkHandler: ProsePlugin = new ProsePlugin({
+  props: {
+    handleClickOn(
+      view: RdfaEditorView,
+      pos: number,
+      node: PNode,
+      nodePos: number,
+      event: MouseEvent
+    ) {
+      const schema = view.state.schema;
+      // The handler only handles clicks on nodes, and the link is a mark of the textnode inside the node, so we have to get it
+      const textNode = node.childAfter(pos);
+      if (!textNode || !textNode.node) return;
```

Comment:
I know I know it's pedantic, and if you disagree you can merge without applying in this case (cause its really _just_ a return)

```suggestion
      if (!textNode || !textNode.node) { 
        return; 
      }
```

Reply:
I prefer your syntax, I don't know what happened :), thanks for catching that